tensorbay/sensor/intrinsics.py

```diff
@@ -362,6 +468,63 @@ class CameraIntrinsics(ReprMixin):
         _distortion_coefficients: It is the deviation from rectilinear projection. It includes
             radial distortion and tangential distortion.
 
+    Example:
+        >>> matrix = [[1, 3, 3],
+        ...           [0, 2, 4],
+        ...           [0, 0, 1]]
+        >>> # Init with _init_distortion is False
+        >>> camera_intrinsics = CameraIntrinsics(matrix, p1=5, k1=6, _init_distortion=False)
+        >>> camera_intrinsics
+        CameraIntrinsics(
+            (camera_matrix): CameraMatrix(
+                (fx): 1,
+                (fy): 2,
+                (cx): 3,
+                (cy): 4,
+                (skew): 3
+            ),
+            (distortion_coefficients): None
+        )
+
+        >>> # Init from camera calibration parameters, skew is optional
+        >>> camera_intrinsics = CameraIntrinsics(fx=1,
+        ...                                  fy=2,
+        ...                                  cx=3,
+        ...                                  cy=4,
+        ...                                  p1=5,
+        ...                                  k1=6)
+        >>> camera_intrinsics
+        CameraIntrinsics(
+            (camera_matrix): CameraMatrix(
+                (fx): 1,
+                (fy): 2,
+                (cx): 3,
+                (cy): 4,
+                (skew): 0
+            ),
+            (distortion_coefficients): DistortionCoefficients(
+                (p1): 5,
+                (k1): 6
+            )
+        )
+
+        >>> # Init from 3x3 sequence array
```

Comment:
Since the first and the third examples use the same matrix, it would be better to place them together.